topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -550,39 +558,50 @@ than a given number of samples. Either a percentage of the total number of sampl
 >     - *"Minimum fraction of samples"*: `0.9`
 >     - *"Width of overlapping m/z slices"*: `0.01`
 >   - *"Get the Peak List"*: `Yes`
->     - *"Replace the remain NA by 0 in the dataMatrix"*: `No`
+>     - *"If NA values remain, replace them by 0 in the dataMatrix"*: `No`
```

Comment:
in the EU server with version version 3.3.4.0 the former (deleted) naming was still present in the tool

Reply:
The renaming was updated in 3.6.1 that was just released this summer - already available on main toolshed 😉